apps/portal/src/styles/index.css

```
@@ -160,6 +170,36 @@
     }
   }
+  @keyframes auth-packet-rise {
+    0%   { transform: translateY(0)     rotate(-4deg); opacity: 0;   }
+    8%   { opacity: 1;                                               }
+    88%  { opacity: 0.7;                                             }
+    100% { transform: translateY(-80px) rotate(5deg);  opacity: 0;  }
+  }
+}
+
+@utility auth-bg-grid {
+  background-image: radial-gradient(circle, hsl(var(--primary) / 0.055) 1.5px, transparent 1.5px);
```

Stylelint (17.7.0)

[error] Expected "0.055" to be "5.5%" (alpha-value-notation)

(alpha-value-notation)